### ISSUE TITLE
Fit one full page to window & fit to window width.

### ISSUE THREAD

**Issue by InfSource** [pvginkel/PdfiumViewer#24]
Hello,

The "Fit page" functionality is very important for us since it is an essential part of any PDF Viewer.

Could you help us with it?

**Comment by pvginkel** [comment]
The `PdfRenderer` class has a `ZoomMode` property which either automatically fits the contents to the maximum age width or height. Can you have a look whether this is what you're looking for?

**Comment by InfSource** [comment]
Yes, it's very similar to what I need, but "fit page" applies only after switching from one mode to another.
Which method should I use for applying "fit page" for the current page?

**Comment by pvginkel** [comment]
I'm not sure what you're looking for. Do you want to introduce something like a toolbar button that the user can press?

**Comment by InfSource** [comment]
Yes, I want to use it for this button

**Comment by pvginkel** [comment]
That explains a lot. I'll have a look at this when I get the chance.

**Comment by pvginkel** [comment]
Please have a look at the demo application. It has two new buttons that do what you're looking for.

**Comment by InfSource** [comment]
Yes, it works well. But there was a problem with FitHeight when the page is Landscape.

**Comment by pvginkel** [comment]
Could you please create a new issue for this? I didn't change the library
for those buttons and it probably also doesn't work if you set the mode of
the viewer to fitwidth.

**Comment by InfSource** [comment]
FitWidth works correctly. Similarly, it works in Acrobat Reader. http://myscreenshot.info/i/0006/265251434377821885379631358467.png

**Comment by pvginkel** [comment]
Thanks. I'll have a look at the issue when I have the chance.
